server/models/index.js

```diff
@@ -1,3 +1,19 @@
+<<<<<<< HEAD
+const {Sequelize} = require('sequelize')
+const {sequelize} = require('../db')
+
+const Sauce = sequelize.define("sauces", {
+  name: Sequelize.STRING,
+  image: Sequelize.STRING,
+});
+
+const Item = sequelize.define('items', {
+  title: Sequelize.STRING,
+  description: Sequelize.STRING,
+  price: Sequelize.STRING,
```

Comment:
I think price should be a number

```suggestion
  price: Sequelize.Integer,
```